dropwizard-metrics/src/main/java/io/dropwizard/metrics/ScheduledReporterManager.java

```diff
@@ -10,16 +10,32 @@
 public class ScheduledReporterManager implements Managed {
     private final ScheduledReporter reporter;
     private final Duration period;
+    private final boolean reportOnStop;
 
     /**
      * Manages the given {@code reporter} by reporting with the given {@code period}.
      *
      * @param reporter the reporter to manage.
-     * @param period the frequency to report metrics at.
+     * @param period   the frequency to report metrics at.
+     * @see #ScheduledReporterManager(ScheduledReporter, Duration, boolean)
+     * @deprecated Please specify the {@code reportOnStop} parameter as well.
      */
+    @Deprecated
```

Comment:
IMO, no need to deprecate just because it's overloaded.

Reply:
Agree. Removed deprecation but kept the `@see` reference